python/GafferUI/SpreadsheetUI/_PlugTableView.py

```diff
@@ -535,6 +536,11 @@ def __prependRowMenuItems( self, menuDefinition, plugs ) :
 		includesDefaultRow = any( [ row.isSame( rowsPlug.defaultRow() ) for row in rowPlugs ] )
 		allLocked = all( [ Gaffer.MetadataAlgo.readOnly( row ) for row in rowPlugs ] )
 
+		# We don't have an item for managing the enabled state of rows, because when a
+		# row is disabled (via Qt.ItemIsEnabled flag), those indices are no longer reported
+		# from the selection model. So you could use the item to turn them off, but its
+		# hard to turn them back on again.
```

Comment:
I'm not entirely convinced by this argument. It seems to be "because it would be hard to enable lots of disabled rows, we're going to make it hard to disable lots of enabled rows".

Reply:
Its more about symmetry. I implemented it, and its incredibly frustrating. You 'learn' there is a menu item to control the enabled state of selected rows. But, 50% of the time, even though the row looks selected, you can't use the menu. Worse than that, if you did get to the menu by creating a contiguous selection from two enabled rows either side of a disabled row, the menu wouldn't change the disabled row's state. It ends up looking more like a broken feature. At least if its not there, you're not set for confusion/disappointment most of the time. Felt like it'd be better to wait for a feature request for it, than constantly explain why its not a bug when it doesn't work how people would intuitively expect it to.